config/initializers/administrate.rb

```diff
@@ -0,0 +1 @@
+Administrate::Engine.add_stylesheet "michigan-benefits/administrate-extensions"
```

Comment:
Link to https://github.com/thoughtbot/administrate/blob/736266dd81f0a1b5ded6a268b93bd7de5c6c1a6f/lib/administrate/engine.rb#L31

Reply:
that is highlighting a line of code that just has an `end` -- typo?

Reply:
I totally forgot you can highlight an entire range:
https://github.com/thoughtbot/administrate/blob/736266dd81f0a1b5ded6a268b93bd7de5c6c1a6f/lib/administrate/engine.rb#L25-L31

I';m going to add them as comments; but like I said there isn't actual docs for this :(